pkg/node/node.go

```diff
@@ -87,11 +87,23 @@ func (n *Node) AnnounceSignerPresence(
 		return nil, fmt.Errorf("failed to set broadcast channel filter: [%v]", err)
 	}
 
+	// TODO: Use generic type for representing address in node.go instead of
+	// common.Address from go-ethereum. The current implementation is not
+	// host-chain implementation agnostic.
+	// To do not propagate ethereum-specific types any further, we convert
+	// addresses to strings before passing them to AnnounceProtocol.
+	keepAddressString := keepAddress.Hex()
+	keepMembersAddressesStrings := make([]string, len(keepAddress))
```

Comment:
not sure if I follow, `keepMembersAddressesStrings` slice needs to be a size of `len(keepAddress)`?

Reply:
Yes, we are transforming `[]common.Address` to `[]string`. In another language, I'd just write `keepAddresses.map(_.Hex())`.

Reply:
this is what I meant https://github.com/keep-network/keep-ecdsa/pull/654#discussion_r549397668 :)